api/src/db/migrations/v106_add_pet_care_housing_essential_question.ts

```diff
@@ -377,3 +378,223 @@ type v106GetFilingResponse = {
   standingDoc: string;
   certifiedDoc: string;
 };
+
```

Comment:
This is interesting: we're adding generator functions to the previous migration as a part of the work for `v107`. This feels like something that should be the responsibility of the author of `v106`. I guess this is mostly copypaste, but if this piece blocks the next migration it feels somewhat unfinished to me if we leave it to the next person to generate.

I'm feeling more strongly about generating a migration generator script.

Reply:
I agree, JP, and was confused as to why that would be the "work" of the person implementing the next migration. I'd love additional feedback.

A migration generator script would be cool.

Reply:
The rationale was that not every migration needs generators - it only needs them if the following migration has a test written. 

We write migrations sometimes even if they don't have logic, like adding a new field that will always have a static value. Those migrations don't typically get tests added.

So if v107 was a migration that didn't need a test, then we'd have no use for generators in v106 - they would be unused functions. Which is generally, something to avoid, in my opinion.

That's the context why (so far) we don't add the generators at the same time as the migration file. Happy to discuss what we prefer in the tradeoff of having-unused-functions-laying-around versus having-to-edit-a-previous-migration-file-when-testing